keylogger.pyw

```diff
@@ -50,24 +51,39 @@ class Keylogger:
 
             key = specialCases.get(key, self.formatSpecialKey(key))
 
-        # finally, add the key name to our global `self.log` variable
```

Comment:
the `self.log` variable is not a global variable.

It's defined inside the class and is not available outside the class

Reply:
I see, during initial stages of developing this, I needed them, now i guess its not neccessary